.github/copilot-instructions.md

```
@@ -70,231 +72,122 @@ globs: *
+            - キュウリ → 食べ物/野菜/きゅうり
+            - ナスビ → 食べ物/野菜/ナス
+            - 揚げ物 → 食べ物/料理/揚げ物
+        - 硬い・柔らかいを階層型カテゴリに変換
+            - 硬い → 素材・材質・生地, 素材・材質・生地/硬い
+            - 硬い → 素材・材質・生地/硬い
+        - サウAkyoをパロディ/人物に修正
+        - 複数のAkyoカテゴリを修正
+            - ハウスネイルAkyo: パロディ/ジャム・ザ・ハウスネイルを追加
+            - まめひなたAkyo: 自然を削除
+            - 胴長パグAkyo: 動物/いぬを追加
+            - エプリAkyo: パロディ/VRChatを追加
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 6; Actual: 12

(MD007, ul-indent)